addon/components/react-component.js

```diff
@@ -1,10 +1,10 @@
-import Ember from 'ember';
-import React from 'npm:react';
-import ReactDOM from 'npm:react-dom';
-
-import getMutableAttributes from 'ember-cli-react/utils/get-mutable-attributes';
-import lookupFactory from 'ember-cli-react/utils/lookup-factory';
+import Ember from "ember";
```

Comment:
Can you switch this file back to single quotes and change to single quotes elsewhere in this PR